mock-builder/src/lib.rs

```diff
@@ -312,16 +284,7 @@ where
 #[macro_export]
 macro_rules! register_call {
 	($f:expr) => {{
-		$crate::register(|| (), $f, CallIds::<T>::insert);
-	}};
-}
-
-/// Register a mock function into the mock function storage for a pallet with
-/// instances. Same as `register()` but it uses as locator who calls this macro.
-#[macro_export]
-macro_rules! register_call_instance {
-	($f:expr) => {{
-		$crate::register(|| (), $f, CallIds::<T, I>::insert);
```

Comment:
No longer need it, because the `I` for the instance is defined optionally in the mocked type.